addons/interact_menu/functions/fnc_keyDown.sqf

```diff
@@ -19,9 +19,21 @@ params ["_menuType"];
 
 if (GVAR(openedMenuType) == _menuType) exitWith {true};
 
+// Conditions: Don't open when editing a text box
+private _isTextEditing = false;
+{
+    if (ctrlType (focusedCtrl _x) == 2) then {
+        _isTextEditing = true;
+        break;
+    };
+} forEach (allDisplays select {!isNull (focusedCtrl _x)});
```

Comment:
I think this can be written using `findIf`. Also, use `CT_EDIT` instead of magic number `2`.